cmd/site-replication.go

```diff
@@ -1834,6 +1872,35 @@ func (c *SiteReplicationSys) syncToAllPeers(ctx context.Context) error {
 				return errSRBucketMetaError(err)
 			}
 		}
+
+		// Replicate ILM expiry rules if needed
+		if addOpts.ReplicateILMExpiry && (meta.lifecycleConfig != nil && meta.lifecycleConfig.HasExpiry()) {
+			var expLclCfg lifecycle.Lifecycle
+			expLclCfg.XMLName = meta.lifecycleConfig.XMLName
+			for _, rule := range meta.lifecycleConfig.Rules {
+				if !rule.Expiration.IsNull() || !rule.NoncurrentVersionExpiration.IsNull() {
```

Comment:
need additional filtering of transition settings if any

Reply:
Ack